_content/essays/love-unknowing-and-female-filth_amy_paris_langenberg.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Love, Unknowing, and Female Filth: The Buddhist Discourse of Birth as a Vector of Social Change for Monastic Women in Premodern South Asia"
+authors:
+  - "Amy Paris Langenberg"
+external_url: "https://www.degruyter.com/document/doi/10.1515/9783110674088-012/pdf"
+source_url: "https://www.degruyter.com/document/doi/10.1515/9783110674088-012/html"
```

Comment:
Let's use the DOI instead where possible 😁 https://doi.org/10.1515/9783110674088-012